### PR TITLE
NoSpawn Entitys

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Added NoSpawn on all doors we want to avoid mapping on ships to cleanup the spawn menu, left only the one we do use.
Added also the windoors, and large APC.
Warp points and switchs

## Why / Balance
Less bloat on spawn menu, less room for errors when making a grid

## Technical details
.yml

## Media
N/A

## Breaking changes
All thing that are mapped will not be effected by this change.

**Changelog**
N/A

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml [hunk 1]
```diff
@@ -900,9 +900,10 @@
   parent: VendingMachineSeedsUnlocked
   id: VendingMachineSeeds
   suffix: Hyroponics
-  components:
```
Comment (Cheackraze) [inline Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml:903]: Why are we removing this?

Comment (dvir001) [reply Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml:903]: Its taking it from the parent so I didnt remove it, also its a copy of the machine so "NoSpawn" so only one exists but mapped one wont be removed or damaged.

Comment (Cheackraze) [reply Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml:903]: gotcha

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/easy_pry.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@
   id: AirlockExternalEasyPry
   suffix: External, EasyPry
   description: It opens, it closes, it might crush you, and there might be only space behind it. Has to be manually activated. Has a valve labelled "TURN TO OPEN"
+  noSpawn: true # Frontier
```
Comment (Cheackraze) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/easy_pry.yml:6]: these i think should be kept in for POI or other things like bluespace events

File: Resources/Prototypes/Entities/Structures/Doors/Airlocks/easy_pry.yml [hunk 3]
```diff
@@ -11,6 +12,7 @@
   id: AirlockExternalGlassEasyPry
   suffix: External, Glass, EasyPry
   description: It opens, it closes, it might crush you, and there might be only space behind it. Has to be manually activated. Has a valve labelled "TURN TO OPEN"
+  noSpawn: true # Frontier
```
Comment (Cheackraze) [inline Resources/Prototypes/Entities/Structures/Doors/Airlocks/easy_pry.yml:15]: same with the other easy pry kind, i think these still can have a use for us

**Review by Cheackraze — COMMENTED** [review]
the easy pry ones i think can stay, i think they can be used in spooky POI maps with no power, otherwise its good

**Pull request closed without merging**